runtime/dancebox/src/lib.rs

```diff
@@ -792,7 +792,19 @@ impl_runtime_apis! {
         }
 
         fn authorities() -> Vec<NimbusId> {
-            pallet_authority_assignment::CollatorContainerChain::<Runtime>::get(Session::current_index())
+
+            // Check whether we need to fetch the next authorities or current ones
```

Comment:
Modified this to actually match what the chain expects, which is, always the set of authorities that has to build the next orchestrator block

Reply:
We have a few integration tests that use this value and they didn't fail after this change, can you add a test for this edge case?

Reply:
they dont use this value, they use the other api (I believe it is `TanssiAuthorityAssignmentApi`). We maintain this api for two reasons though:
- because we are still using aura, and it might be expected by certain tools (i.e., chopsticks)
- because it implementes the slot_duration function (which we could also move to the other runtime api, but I think it does not hurt to maintain this on)

Reply:
I updated this one because chopsticks has to be a bit more generic, and therefore if it detects that aura consensus is being used, it queries this api